controllers/storage/minio.go

```diff
@@ -41,21 +51,30 @@ func (m *MinIOReconciler) Reconcile() (*lcm.CRStatus, error) {
 		return m.ProvisionExternalStorage()
 	}
 
-	err := m.KubeClient.Get(m.Ctx, m.getminIONamespacedName(), &minioCR)
+	err := m.KubeClient.Get(m.getMinIONamespacedName(), &minioCR)
 	if k8serror.IsNotFound(err) {
-		// TODO need test
 		return m.Provision()
 	} else if err != nil {
-		return minioNotReadyStatus(ErrorReason0, err.Error()), err
+		return minioNotReadyStatus(GetMinIOError, err.Error()), err
+	}
+
+	var minioStatefulSet appsv1.StatefulSet
+	err = m.KubeClient.Get(m.getMinIONamespacedName(), &minioStatefulSet)
+	if err != nil {
+		return minioNotReadyStatus(GetMinIOError, err.Error()), err
 	}
 
+	if minioStatefulSet.Status.ReadyReplicas == m.HarborCluster.Spec.Storage.InCluster.Spec.Replicas {
+		// TODO create default bucket
+		return m.ProvisionInClusterSecretAsOss(&minioCR)
```

Comment:
What does `ProvisionInClusterSecretAsOss` means? Why `AsOss`?

Reply:
minIO implements both interfaces of Oss and S3, so we can create in-cluster secret as Oss or s3.